src/models/artboard/README.md

```diff
@@ -0,0 +1,66 @@
+# ArtBoard Model
+
+ArtBoard Model is responsible for storing the structure of the class and its parameter used for artBoardDraw funtionality.These classes are called by the `ArtBoardSketch.tsx` and `ArtBoardTurtle.tsx` There are majorly 4 different classes.
```

Comment:
The ArtBoard Model is responsible for storing the structure of the class and the parameters used by the artBoardDraw function. There are four classes in the model, whcih are called by the `ArtBoardSketch.tsx` and `ArtBoardTurtle.tsx`.